utils/upload_utils.py

```diff
@@ -20,31 +22,27 @@ def post_texts_with_geo(
 
         inputs = []
         for idx, text in enumerate(text_batch):
-            inp = resources_pb2.Input(
-                data=resources_pb2.Data(
-                    text=resources_pb2.Text(raw=text),
-                    geo=resources_pb2.Geo(
-                        geo_point=resources_pb2.GeoPoint(
-                            longitude=geo_points_batch[idx]["lon"],
-                            latitude=geo_points_batch[idx]["lat"],
-                        )
-                    ),
-                )
-            )
-
-            ParseDict(metadata_batch[idx], inp.data.metadata)
-            inputs.append(inp)
-
-        post_inputs_request = service_pb2.PostInputsRequest(
-            user_app_id=userDataObject, inputs=inputs
-        )
-
-        post_inputs_response = stub.PostInputs(post_inputs_request)
-        # print(response)
-        if post_inputs_response.status.code != status_code_pb2.SUCCESS:
-            raise Exception("PostInputs request failed: %r" % post_inputs_response)
+            input_obj = Inputs(logger_level="ERROR")
+            id = hashlib.md5(text.encode("utf-8")).hexdigest()
```

Comment:
```suggestion
            id = uuid.uuid4().hex
```
this should be random, not depend on text as for same text and diff geo locations. Id remains same and `PostInputs` throws an error.